source/fare/fare.cpp

```diff
@@ -67,8 +72,14 @@ static Label next_label(Label label, Ticket ticket, const SectionKey& section) {
         // empty ticket, it is juste a change
         // we have to update the number of changes and the duration with the same ticket
         if (ticket.caption == "" && ticket.value == 0) {
+#if (FARE_LOG_ENABLED)
+            LOG4CPLUS_TRACE(logger, "Empty ticket, free change");
+#endif
```

Comment:
Ok, the macro has been removed in 24eb4dcc95284c5210af415bf8c2338ba80477b5
I introduced it to avoid checking the condition if(log_level > trace) when the log is disabled, but it makes the code harder to read anyway